src/v2/Apps/Purchase/Utils/orderHelper.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  CheckCircleFillIcon,
+  PendingCircleIcon,
+  XCircleIcon,
+} from "@artsy/palette"
+import React from "react"
+
+export const getOrderIcon = status => {
+  switch (status) {
+    case "submitted":
+      return <PendingCircleIcon fill="black60" />
+    case "canceled":
+      return <XCircleIcon fill="red100" />
+    case "refunded":
+      return <XCircleIcon fill="red100" />
+    case "approved":
+      return <CheckCircleFillIcon />
+    case "fulfilled":
+      return <CheckCircleFillIcon />
+  }
+}
+
+export const getOrderColor = status => {
+  switch (status) {
+    case "submitted":
+      return "black60"
+    case "canceled":
+      return "red100"
+    case "refunded":
+      return "red100"
+    case "approved":
+      return "black100"
+    case "fulfilled":
+      return "black100"
```

Comment:
**suggestion-non-blocking :** use fall through to avoid repeating returns

Instead of doing something like 
```
    case "canceled":
      return "red100"
    case "refunded":
      return "red100"
```
it can be simplified into 
```
    case "canceled":
    case "refunded":
      return "red100"
```

Reply:
🤯 Didn't know about this. So cool. https://github.com/artsy/force/pull/6436/commits/b7ff8540581ed57aac2cd6198860f9144e0dac0a